### DISCUSSION TITLE
Custom field labels

### DISCUSSION THREAD

**Discussion by antonio8484** [snipe/snipe-it#9139] [Q&A]
Hi, I'm new to Snipe-IT so chances are that I missed something in the guides.

I have set up a few custom fields, and some of them have regex validation. The issue I have is that these fields look really ugly when I add a new asset, because they pretty much do a copy and paste of the condition, like 'Enter regex:xxxxxxxxxxx'.
Since some of them are quite long and look quite confusing, is there any option to edit the 'Enter .....' label, or maybe even to remove it completely and just leave a blank text box?

**Comment by snipe** [comment]
Can you just use the help text option, which lets you add helpful text aimed at guiding your admins/users? We can't really remove the placeholder text (the one that's asking for a regex because _some_ people will understand what that means and they want that reminder.

**Reply by antonio8484** [reply]
My issue is that it looks quite ugly having to see all the checks that the system does behind the scenes, even for simple fields.
For example, I use regex to check that a Bitlocker key is correct. I can set the help text to something like "Please use the following pattern: XXXXXX-XXXXXX", but to have the veeery long regex rule showing up all the time is, again, quite ugly and unnecessary (in my opinion).
It would be really nice if that bit could be customised, but I understand that it may require too much work.

EDIT: Just to expand a little bit more. What I would like to have is something like the MAC type in custom fields: all it says is 'Enter mac text', it doesn't show the full check that the system does, which, to a user, would be 99.99% of the time irrelevant anyway.
